src/components/DashBoard/ManageCountries/delete/DeleteModal.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { toast } from "react-toastify";
+
+//===MUI IMPORTS===
+import { Box, Button, Modal } from "@material-ui/core";
+//===COMPONENTS IMPORTS===
+import classes from "./ModalComponent.module.css";
+import { useDispatch } from "react-redux";
+// import {
+//   DeleteCountry,
+//   fetchAllCountrys,
+// } from "../../../../store/Actions/CountryActions";
+import { DeletePost } from "../../../../store/Actions/PostActions";
+import { DAV_APIS } from "../../../../Adapter";
+
+const DeleteModal = (props) => {
+  const { open, setOpen, Id, setSearchTerm, source, language } = props;
+  const [loading, setLoading] = useState(false);
+  const dispatch = useDispatch();
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+  const style = {
+    bgcolor: "background.paper",
+  };
+
+  const deleteHandler = async () => {
+    setLoading(true);
+    if (source === "country") {
+      await DAV_APIS.deleteCountryById(Id);
+      toast.success("Country deleted Successfully");
+      // dispatch(fetchAllCountrys());
```

Comment:
Remove any redux related code that we have commented out and not using. This includes the slices, actions and any other file or line of code that you are sure that we are not using any more